migrations/44-50/removed-backward-incompatibility.md

```diff
@@ -37,3 +37,36 @@ class MyModel extends ListModel {
 
 ### CSS removals
 The CSS class ".ie11" was removed [via PR #39018](https://github.com/joomla/joomla-cms/pull/39018)
+
+### Language strings
+File: administrator/language/en-GB/com_admin.ini
+Strings: COM_ADMIN_A11Y_SETTINGS_FIELDSET_LABEL, COM_ADMIN_A11Y_SETTINGS_FIELD_CONTRAST, COM_ADMIN_A11Y_SETTINGS_FIELD_FONTSIZE, COM_ADMIN_A11Y_SETTINGS_FIELD_HIGHLIGHT, COM_ADMIN_A11Y_SETTINGS_FIELD_MONOCHROME
+
+File: administrator/language/en-GB/com_media.ini 
+Strings: COM_MEDIA_FIELD_ILLEGAL_MIME_TYPES_DESC, COM_MEDIA_FIELD_ILLEGAL_MIME_TYPES_LABEL
+
+File: administrator/language/en-GB/com_newsfeeds.ini
+Strings: COM_NEWSFEEDS_NEWSFEEDS, COM_NEWSFEEDS_SAVE_SUCCESS, COM_NEWSFEEDS_SEARCH_IN_TITLE, COM_NEWSFEEDS_SELECT_FEED, COM_NEWSFEEDS_SUBMENU_CATEGORIES, COM_NEWSFEEDS_SUBMENU_NEWSFEEDS
```

Comment:
```suggestion
Strings: COM_NEWSFEEDS_SEARCH_IN_TITLE, COM_NEWSFEEDS_SELECT_FEED, COM_NEWSFEEDS_SUBMENU_CATEGORIES, COM_NEWSFEEDS_SUBMENU_NEWSFEEDS
```
please check https://github.com/joomla/joomla-cms/pull/40493 and https://github.com/joomla/joomla-cms/pull/40533